examples/rivets/index.html

```diff
@@ -0,0 +1,123 @@
+<!doctype html>
+<html lang="en" data-framework="rivets">
+	<head>
+		<meta charset="utf-8">
+		<title>Rivets • TodoMVC</title>
+		<link rel="stylesheet" href="node_modules/todomvc-common/base.css">
+		<link rel="stylesheet" href="node_modules/todomvc-app-css/index.css">
+	</head>
+	<body>
+		<script type="text/template" id="todos">
+			<header id="header">
```

Comment:
https://github.com/tastejs/todomvc/pull/1304/files#diff-34eab039025c337bc6534509702dbb78R18

see how we use classes here?

and the 2.0 app css
https://github.com/tastejs/todomvc/pull/1304/files#diff-e6462df4cdf27b57d669e5c8a2dba09fR18

:) that would be ideal for this implementation